python/mxnet/gluon/nn/basic_layers.py

```diff
@@ -442,3 +442,90 @@ def hybrid_forward(self, F, x):
 
     def __repr__(self):
         return self.__class__.__name__
+
+
+class InstanceNorm(HybridBlock):
+    """
+    Applies instance normalization to the n-dimensional input array.
+    This operator takes an n-dimensional input array where (n>2) and normalizes 
+    the input using the following formula:
+    .. math::
+
+    out = \frac{x - mean[data]}{ \sqrt{Var[data]} + \epsilon} * gamma + beta
+    This layer is similar to batch normalization layer (`BatchNorm`)
```

Comment:
A new line here too, since the block of source code is supposed to be surrounded by empty lines.

Reply:
Does it look okay now? I am not familiar with rst